ISISComputingGroup/ibex_gui
# Ticket972 synoptic pvs all remote

https://github.com/ISISComputingGroup/IBEX/issues/971

To test:
* The client now does not allow local/remote to be set, and does not add PVType to the XML on the BlockServer
* The client ignores the PVType flag set on old synoptics. No problems should occur if a local PV is set, other than it can not connect to that PV until the prefix is added.

**Note: Larmor has still not been updated. Should we do this before merging?**
